### PR TITLE
in-toto doc assessment

### PR DESCRIPTION
**Pull request by jbogarthyde**
resolves: #163

### CODE REVIEW

**Comment by nate-double-u** [comment]
@thisisobate, could you help @jbogarthyde with the website/technology part of this assessment like you're helping David with the other one?

File: assessments/0009-iin-toto.md [hunk 1]
```diff
@@ -0,0 +1,292 @@
...
+
+- The doc roadmap should also list and link reference docs for other implementations, such as auto-generated docs for the Go implementation: https://pkg.go.dev/github.com/in-toto/in-toto-golang. (This doc is not at all parallel with the Python RTD reference doc - it is all in GitHub, and has no introductory or explanatory content, or navigational aids.)
+
+- As noted below, policy and procedures for documenting implementations need to be decided and published for contributors.
+
+**New user content**
+
+- The [README for the main repo](https://github.com/in-toto/in-toto) looks like the intended starting place. This content should be surfaced as a separate **"Getting Started" document**, with a prominent pointer on the home page, such as the top menu item in the "Get Started" menu.
+
+We recommend moving all of this the doc to a documentation platform such as RTD. Put some brief introductory notes in the README that points the user to the documentation.
```
Comment (nate-double-u) [inline assessments/0009-iin-toto.md:109]: We should clarify what we mean by platform.

Generally the TechDocs team suggests a stack of:
- Hugo
- Netlify
- Docsy

I think that read the docs fits here, but we should try to figure out how it all works together in an architecture suggestion.

File: assessments/0009-iin-toto.md [hunk 2]
```diff
@@ -0,0 +1,292 @@
...
+  
+- Determine whether the Tech Overview currently in the spec is suitable for new users or for contributors, and adapt or create another version of suitable depth for the other audience. The most in-depth overview should also point to academic papers for further architectural detail:
+https://www.usenix.org/system/files/sec19-torres-arias.pdf 
+https://www.usenix.org/conference/usenixsecurity19/presentation/torres-arias 
+
+- The new Doc home page should probably be the landing page for the [read-the-docs site](https://in-toto.readthedocs.io/en/latest/), which currently lands on the auto-generated Python reference doc. Tech overviews addressed to specific audiences should be added to RTD as text-only pages, organized by intended audience. 
+
+**Reference docs** for different implementations
+
+- Decide where reference docs for different implementations should live in the doc structure and where their sources live in the repo structure.
```
Comment (nate-double-u) [inline assessments/0009-iin-toto.md:97]: We can make an overall recommendation that the documentation currently living in the various github repos should be brought to the docs section of the website as the "source of truth" set of documentation.

File: assessments/0009-iin-toto.md [hunk 3]
```diff
@@ -0,0 +1,292 @@
...
+
+- Include a [System Overview](https://github.com/in-toto/docs/blob/master/in-toto-spec.md#2-system-overview) tailored to new users as the intro to new users.
+
+- The [Glossary](https://github.com/in-toto/docs/blob/master/in-toto-spec.md#17-terminology) should be surfaced as a separate document and formatted in an alphabetized table for easy reference.
+
+- The [Workflow Description](https://github.com/in-toto/docs/blob/v1.0/in-toto-spec.md#51-workflow-description) in the spec identifies the different user roles. Adapt this to a separate document that also points to the appropriate doc for each role, which would be particularly helpful to new users.
+
+**Content maintainability**
+
+Almost all doc content is in GitHub (in README files, the Specification, or comments in demos). Most of it should be surfaced to indexed documents on the website to make it versionable, editable, and localizable.
```
Comment (nate-double-u) [inline assessments/0009-iin-toto.md:121]: +1

I think this maybe a toplevel suggestion (and I referenced this above.)

File: assessments/0009-iin-toto.md [hunk 4]
```diff
@@ -0,0 +1,292 @@
...
+
+- 1 = (Is not present or requires significant work)
+- 3 = (Is present, but needs work)
+- 5 = (Is executed extremely well or no improvement required)
+
+**Comments**
+
+- **Communication methods documented:** [Contact channels list](https://in-toto.io/contact/) available from home page under **Community > Contact**.
+
+- **Beginner friendly issue backlog:** "Good First Issue" label is available and used for PRs for specific implementations. Not in much general use.
```
Comment (nate-double-u) [inline assessments/0009-iin-toto.md:157]: We should take this opportunity to surface a new tool we've built: https://clotributor.dev/, https://github.com/cncf/clotributor

"One of the CLOTributor's goals is to surface interesting opportunities for potential contributors to Cloud Native projects, allowing them to find those that suit their skills and interests best.

To achieve this, CLOTributor scans periodically hundreds of repositories, indexing issues that match certain criteria:

    Contain the help wanted label
    Their state is OPEN
    They are unassigned
    Updated within the last year
"

File: assessments/0009-iin-toto.md [hunk 5]
```diff
@@ -0,0 +1,292 @@
...
+
+- **“New contributor” getting started content:** [Community README](https://github.com/in-toto/community/blob/main/README.md) points to "Good First Issues" list for specific repos. Can't find explicit instructions for new contributors, or any first issues for other kinds of contribution (such as docs).
+
+- **Project governance documentation:** Clearly described and discoverable on [GOVERNANCE page](https://github.com/in-toto/community/blob/main/GOVERNANCE.md).
+
+**RECOMMENDATIONS**
+
+**Communication methods documented**
+
+- Since most of the documentation is in GitHub, rather than on the web site, the [Contact channels list](https://in-toto.io/contact/) could be usefully added to the READMEs for the [main](https://github.com/in-toto/in-toto) and [community](https://github.com/in-toto/community) repos.
```
Comment (nate-double-u) [inline assessments/0009-iin-toto.md:167]: Should this be surfaced onto the website? I know that things like CONTRIBUTING often live in the repo, but there may be value in having these sorts of things on the website (I'm not too concerned about dulpication in this case as things are not likely to change once set.)

File: assessments/0009-iin-toto.md [hunk 6]
```diff
@@ -0,0 +1,292 @@
...
+- **HTTPS access & HTTP redirect:** HTTPS is the default, HTTP redirects correctly.
+
+- **Intra-site / local search:** No site map or search mechanism, only navigation is through a minimal menu bar.
+
+_Provide comments for each rating above, 1-2 sentences max, bullet point list_
+_Include a list of the top 404s, as reported through analytics or a search console._
+
+**Recommendations**
+
+_Provide a list of recommendations to improve in this area_
```
Comment (nate-double-u) [inline assessments/0009-iin-toto.md:226]: Let me take a closer look at that PR and I'll circle around back to this one. On first glance, it looks like a good idea.

**Review by nate-double-u — COMMENTED** [review]
@jbogarthyde, this is really good, thank you so much.

I think the main shift we should make is around the info architecture recommendations, and how best to communicate them. This maybe a conversation between all of us so that we are consistent.

I've made some comments in line as well, though I've only been able to spend an hour or so thus far, so i may have some more comments moving forward. 

Thanks so much for all your work so far!

**Comment by thisisobate** [comment]
cc: @jbogarthyde

**Comment by nate-double-u** [comment]
Some general feedback that came up in the last in-toto community meeting. Some of this may be applicable to more than just this assessment.

* Actionable feedback is key
* Provide guidelines, exemplars from other projects (We do this, or have a section for it, but may want to surface these kinds of suggestions better)
  * Here are some things that you don’t have, but which CNCF projects should consider
  * An idea came up to provide an executive summary style introduction that lists the top actionable suggestions
    * “easy lifts”
  * Checklists not as valuable as examples
* The community may take better to sectioned, actionable, feedback, rather than a grade on specific sections out of 5
  * When ready, it may be best to parcel out the feedback to specific sections, one at a time

@psgustafson @dwelsch-esi - pinging you as you attended and I don’t want to have missed any important bits of feedback about the process.

File: assessments/0009-iin-toto.md [hunk 7]
```diff
@@ -0,0 +1,375 @@
...
+
+- A basic intro, possibly suitable for evaluators, is already surfaced directly on the [home page About tab](https://in-toto.io/in-toto/).
+- The About page has a link to the [latest spec in GitHub](https://github.com/in-toto/docs/blob/master/in-toto-spec.md), which contains a more comprehensive overview. This intro should immediately be surfaced as a Technical Overview document in RTD. It could be linked to the basic one as "Read more...".
+   (The [PDF link to the stable spec](https://github.com/in-toto/docs/blob/v1.0/in-toto-spec.pdf) is broken. This should be fixed or removed.)
+  
+- Determine whether the Tech Overview currently in the spec is suitable for new users or for contributors, and adapt or create another version of suitable depth for the other audience. The most in-depth overview should also point to academic papers for further architectural detail:
+https://www.usenix.org/system/files/sec19-torres-arias.pdf 
+https://www.usenix.org/conference/usenixsecurity19/presentation/torres-arias 
+
+3. **Reference material**
```
Comment (dwelsch-esi) [inline assessments/0009-iin-toto.md:83]: Duplicate item number 3. Renumber from here starting with 4.

File: assessments/0009-iin-toto.md [hunk 8]
```diff
@@ -0,0 +1,375 @@
...
+
+Detailed **Assessments** and corresponding specific recommendations are divided into three sections:
+
+- **Project documentation:** for end users of the project; aimed at people who intend to use it
+- **Contributor documentation:** for new and existing contributors to the project
+- **Website:** branding, website structure, and maintainability
+
+   Each section rates content based on different [criteria](criteria.md).
+
+## Overall Recommendations
```
Comment (dwelsch-esi) [inline assessments/0009-iin-toto.md:29]: This section ("Overall Recommendations") seems redundant with the final Recommendations section already in the document. Could these two sections be consolidated? I think a discussion of the purpose behind "actionable items" would be appropriate to add to the "How this document works" section, and the actionable items themselves moved to the Recommendations section.

**Comment by jbogarthyde** [comment]
Replacing with new iteration

**Pull request closed without merging**